modules/ducktests/tests/docker/ducker-ignite

```diff
@@ -258,7 +258,7 @@ docker_run() {
     must_do -v docker run --privileged \
         -d -t -h "${node}" --network ducknet "${expose_ports}" \
         --memory=${docker_run_memory_limit} --memory-swappiness=1 \
-        -v "${ignite_dir}:/opt/ignite-dev" --name "${node}" -- "${image_name}"
+        -v "${ignite_dir}:/opt/ignite-dev:delegated" --name "${node}" -- "${image_name}"
```

Comment:
_delegated_ could be misleading. Containers are reused for running tests with ./run_tests.sh script. But docker docs says:

> If changes to the mount source directory are present on the host file system, those changes may be lost when the delegated mount synchronizes with the host source directory.

Check this [article](https://docs.docker.com/docker-for-mac/osxfs-caching/)

Reply:
Possible solution is to force clean up all containers after tests have finished

Reply:
Seems, we have to find a good fix here.
Left's restart this as a separate PR.

Reply:
is this MacOs only issue?

Reply:
Hi! delegated hack is used only for Ignite code. The same time work directory is located under `/mnt/service` directory that is not a volume. Also docker uses different filesystem for non-volumes (by default it is overlayfs) than bare metal. This FS allows native performance but in some circumstances performance could be different (check this [note](https://docs.docker.com/storage/storagedriver/overlayfs-driver/#performance-best-practices)).

So one should not make assumptions about performance based on results in docker only. But this more about heavy benchmarks. Testing in docker helps to find programming issues that visibly slow down some operations.

Reply:
Also, I've found that there is kibosh used in Dockerfile. I'm not confident how exactly it works, may be there are some penalties too (actually I don't know).